diff -up lesspipe-1.85/configure.odt2xt~ lesspipe-1.85/configure
--- lesspipe-1.85/configure.odt2xt~	2020-08-25 09:24:11.000000000 +0200
+++ lesspipe-1.85/configure	2021-04-09 15:45:16.485605866 +0200
@@ -81,10 +81,9 @@ test:
 install:
 	mkdir -p \$(DESTDIR)\$(PREFIX)/bin
 	mkdir -p \$(DESTDIR)\$(PREFIX)/share/man/man1
-	cp ./code2color ./sxw2txt ./tarcolor ./lesspipe.sh \$(DESTDIR)\$(PREFIX)/bin
+	cp ./code2color ./tarcolor ./lesspipe.sh \$(DESTDIR)\$(PREFIX)/bin
 	cp ./lesspipe.1 \$(DESTDIR)\$(PREFIX)/share/man/man1
 	chmod 0755 \$(DESTDIR)\$(PREFIX)/bin/lesspipe.sh
-	chmod 0755 \$(DESTDIR)\$(PREFIX)/bin/sxw2txt
 	chmod 0755 \$(DESTDIR)\$(PREFIX)/bin/code2color
 	chmod 0755 \$(DESTDIR)\$(PREFIX)/bin/tarcolor
 clean:
diff -up lesspipe-1.85/contrib/bsdtar.patch.odt2xt~ lesspipe-1.85/contrib/bsdtar.patch
--- lesspipe-1.85/contrib/bsdtar.patch.odt2xt~	2020-08-25 09:24:11.000000000 +0200
+++ lesspipe-1.85/contrib/bsdtar.patch	2021-04-09 15:45:16.485605866 +0200
@@ -133,7 +133,7 @@
 @@ -664,6 +730,10 @@
    elif [[ "$1" = *OpenDocument\ [CHMPST]* || "$1" = *OpenOffice\.org\ 1\.x\ [CIWdgpst]* ]]; then
      echo "==> append $sep to filename to view the OpenOffice or OpenDocument source"
-     istemp sxw2txt "$2"
+     istemp odt2txt "$2"
 +#elif o3tohtml
 +  elif [[ "$PARSEHTML" = yes && -n "$bsdtarcmd" && ("$1" = *OpenDocument\ [CHMPST]* || "$1" = *OpenOffice\.org\ 1\.x\ [CIWdgpst]*) ]]; then
 +    echo "==> append $sep to filename to view the OpenOffice or OpenDocument source"
diff -up lesspipe-1.85/lesspipe.sh.in.odt2xt~ lesspipe-1.85/lesspipe.sh.in
--- lesspipe-1.85/lesspipe.sh.in.odt2xt~	2021-04-09 15:45:16.485605866 +0200
+++ lesspipe-1.85/lesspipe.sh.in	2021-04-09 15:46:36.771999848 +0200
@@ -1127,19 +1127,19 @@ isfinal() {
     if cmd_exist o3tohtml; then
       msg "append $sep to filename to view the OpenOffice or OpenDocument source"
       istemp "unzip -avp" "$2" content.xml | o3tohtml | parsehtml -
-    elif cmd_exist sxw2txt; then
+    elif cmd_exist odt2txt; then
       msg "append $sep to filename to view the OpenOffice or OpenDocument source"
-      istemp sxw2txt "$2"
+      istemp odt2txt "$2"
     else
-      msg "install at least sxw2txt from the lesspipe package to see plain text in openoffice documents"
+      msg "install at least odt2txt to see plain text in openoffice documents"
     fi
 #elif unzip
   elif [[ "$PARSEHTML" = yes && "$1" = *OpenOffice\ Document* ]] && cmd_exist unzip; then
-    if cmd_exist sxw2txt; then
+    if cmd_exist odt2txt; then
       msg "append $sep to filename to view the OpenOffice or OpenDocument source"
-      istemp sxw2txt "$2"
+      istemp odt2txt "$2"
     else
-      msg "install at least sxw2txt from the lesspipe package to see plain text in openoffice documents"
+      msg "install at least odt2txt to see plain text in openoffice documents"
     fi
 #endif
 #ifdef isoinfo
diff -up lesspipe-1.85/Makefile.odt2xt~ lesspipe-1.85/Makefile
--- lesspipe-1.85/Makefile.odt2xt~	2020-08-25 09:24:11.000000000 +0200
+++ lesspipe-1.85/Makefile	2021-04-09 15:45:16.485605866 +0200
@@ -11,10 +11,9 @@ test:
 install:
 	mkdir -p $(DESTDIR)$(PREFIX)/bin
 	mkdir -p $(DESTDIR)$(PREFIX)/share/man/man1
-	cp ./code2color ./sxw2txt ./tarcolor ./lesspipe.sh $(DESTDIR)$(PREFIX)/bin
+	cp ./code2color ./tarcolor ./lesspipe.sh $(DESTDIR)$(PREFIX)/bin
 	cp ./lesspipe.1 $(DESTDIR)$(PREFIX)/share/man/man1
 	chmod 0755 $(DESTDIR)$(PREFIX)/bin/lesspipe.sh
-	chmod 0755 $(DESTDIR)$(PREFIX)/bin/sxw2txt
 	chmod 0755 $(DESTDIR)$(PREFIX)/bin/code2color
 	chmod 0755 $(DESTDIR)$(PREFIX)/bin/tarcolor
 clean:
