diff -up lesspipe-1.85/lesspipe.sh.in.o3read~ lesspipe-1.85/lesspipe.sh.in
--- lesspipe-1.85/lesspipe.sh.in.o3read~	2021-04-09 15:43:18.342631926 +0200
+++ lesspipe-1.85/lesspipe.sh.in	2021-04-09 15:44:25.672433748 +0200
@@ -1123,7 +1123,7 @@ isfinal() {
     ppthtml "$2" | parsehtml -
 #endif
 #ifdef o3tohtml,unzip
-  elif [[ "$PARSEHTML" = yes && "$1" = *OpenDocument* ]] && cmd_exist unzip; then
+  elif [[ "$PARSEHTML" = yes && "$1" = *OpenDocument* ]] && cmd_exist unzip && cmd_exist o3read; then
     if cmd_exist o3tohtml; then
       msg "append $sep to filename to view the OpenOffice or OpenDocument source"
       istemp "unzip -avp" "$2" content.xml | o3tohtml | parsehtml -
